dev/com.ibm.ws.security.spnego/resources/com/ibm/ws/security/spnego/internal/resources/SpnegoMessages.nlsprops

```diff
@@ -113,3 +113,7 @@ SPNEGO_CAN_NOT_GET_DELEGATE_SERVICE_SPN.useraction=Verify the service principal
 SPNEGO_DELEGATE_SPN_LOGIN_TO_KDC_FAILURE=CWWKS4322E: Delegate SPN {0} can not authenticate to the Key Distribution Center (KDC) using the Kerberos configuration file {1} and keytab file {2}. Login exception was received: {3}
 SPNEGO_DELEGATE_SPN_LOGIN_TO_KDC_FAILURE.explanation=Can not authenticate to the Key Distribution Center (KDC).
 SPNEGO_DELEGATE_SPN_LOGIN_TO_KDC_FAILURE.useraction=Verify the service principal name format, and make sure it exists on the Windows domain and that the Kerberos keytab file contains the service principal name.
+
+SPNEGO_CONFLICTING_SETTINGS_CWWKS4323E=CWWKS4323E: The attribute {0} from the {1} element conflicts with the attribute {2} from the {3} element. Only specify a value on either the {1} or the {3} element. It is suggested to only specify the value on the {1} element.
```

Comment:
Change,
"The attribute {0} from the {1} element conflicts with the attribute {2} from the {3} element. Only specify a value on either the {1} or the {3} element. It is suggested to only specify the value on the {1} element."
to,
"The attribute {0} from the {1} element conflicts with the attribute {2} from the {3} element. Specify only a value on either the {1} or the {3} element. It is suggested to specify only the value on the {1} element."

Reply:
I would suggest instead
""The attribute {0} of the {1} element conflicts with the attribute {2} from the {3} element. Specify a value on either the {1} or the {3} element, but not on both. It is suggested to specify the value on the {1} element."
Also, you need to decide whether the proper nouns (replacement parms) should proceed or follow nouns (and if IIRC the guidelines say it should proceed like this "The foobar attribute of the barfoo element") consistently.

Reply:
SPN needs to be exploded similar to KDC.  Useraction also refers to Windows domain which would not apply to *nix envs.